frontend/src/pages/setupVault/SetupVaultView.tsx

```diff
@@ -88,18 +78,63 @@ const TabbedContent: React.FC = () => {
   };
 
   // screens
-  // 0 - vault setup view
-  // 1 - vault name setup
-  // 2 - keygen peer discovery screens
-  // 3 - keygen verify
-  // 4 - keygen view
-  // 5 - keygen done
-  // 6 - keygen error
+  // 0 - vault setup initial view
+  // 1 - vault setup view
+  // 2 - vault name setup
+  // 3 - keygen peer discovery screens
+  // 4 - keygen verify
+  // 5 - keygen view
+  // 6 - keygen done
+  // 7 - keygen error
   const screens = [
```

Comment:
can you create components for each screen?

Reply:
fixed